website/project/views/contributor.py

```diff
@@ -207,6 +207,10 @@ def project_before_remove_contributor(auth, node, **kwargs):
         if auth.user != contributor:
             raise HTTPError(http.FORBIDDEN)
 
+    if len(node.visible_contributor_ids) == 1 \
+            and node.visible_contributor_ids[0] == contributor._id:
+        raise HTTPError(http.FORBIDDEN)
```

Comment:
Always pass a useful error message in error responses, i.e. pass message_short and/or message_long in the `data` argument to describe the error.